Lib/calendar.py

```diff
@@ -159,8 +159,8 @@ def weekday(year, month, day):
 
 
 def monthrange(year, month):
-    """Return weekday (0-6 ~ Mon-Sun) and number of days (28-31) for
-       year, month."""
+    """Return day 1 weekday (0-6 ~ Mon-Sun) and number of days (28-31)
+    for year, month."""
```

Comment:
"day 1 weekday" is a bit opaque, shall we be more explicit, and closer to docs?

```suggestion
    """Return weekday of first day of month (0-6 ~ Mon-Sun) and number of days (28-31)
    for year, month."""
```
Re-wrapping:
```suggestion
    """Return weekday of first day of month (0-6 ~ Mon-Sun)
       and number of days (28-31) for year, month."""
```